api/apps/geoprocessing/src/marxan-sandboxed-runner/adapters/solutions-output/solutions-output.service.ts

```diff
@@ -30,10 +33,19 @@ export class SolutionsOutput implements Cancellable {
    * Treat this class as coordinator
    *
    */
-  async saveFrom(workspace: Workspace, scenarioId: string): Promise<void> {
+  async saveFrom(
+    workspace: Workspace,
+    scenarioId: string,
+    stdOutput: string[],
+    stdErr?: string[],
+  ): Promise<void> {
     if (!existsSync(workspace.workingDirectory + `/output/output_sum.csv`)) {
       throw new Error(`Output is missing from the marxan run.`);
     }
+    await this.metadataRepository.save(scenarioId, workspace, {
+      stdOutput,
+      stdErr,
+    });
```

Comment:
according to my previous comment, the workspace path could be passed here instead of entire workspace